sdk/labservices/Microsoft.Azure.Management.LabServices/src/Microsoft.Azure.Management.LabServices.csproj

```diff
@@ -4,12 +4,12 @@
 	<!-- Please do not move/edit code below this line -->
 
 	<PropertyGroup>
-		<Version>4.0.0</Version>
+		<Version>4.1.0</Version>
 		<Description>Microsoft.Azure.Management.LabServices</Description>
 		<AssemblyName>Microsoft.Azure.Management.LabServices</AssemblyName>
 		<PackageId>Microsoft.Azure.Management.LabServices</PackageId>
 		<PackageTags>LabServices;ManagedLabs;</PackageTags>
-		<PackageReleaseNotes>Release api version 2021-11-15-preview.</PackageReleaseNotes>
+		<PackageReleaseNotes>Minor Version Update - Bugs Fixed:LabServicesClient constructors are now public</PackageReleaseNotes>
```

Comment:
preview version of swagger spec can't be used to release a stable version of sdk

Reply:
Hi @ArthurMa1978 , Can you please share more details about the comment? We currently have only the preview version, we are yet to release the GA version. Thanks.